keyboards/tweetydabird/lotus58/blok/config.c

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 Markus Knutsson (@TweetyDaBird)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#define I2C_DRIVER I2CD0
+//using raw pin values, blok converter doesn't catch this
+#define I2C0_SDA_PIN 17U
+#define I2C0_SCL_PIN 16U
+#define I2C0_CLOCK_SPEED  400000
+
+#define SERIAL_USE_MULTI_TRANSACTION
+#define SPLIT_MODS_ENABLE
+#define SPLIT_TRANSPORT_MIRROR
```

Comment:
```suggestion
```
These options are already enabled in `info.json` at the higher level (`split.transport.sync_modifiers` and `split.transport.sync_matrix_state`); no need to repeat them here.